plugins/modules/podman_system_info.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+# Copyright (c) 2025 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+module: podman_system_info
+author:
+  - Johnson Lien (@johnsonlien)
+short_description: Get podman system information from host machine
+description: Runs "podman system info --format json" on host machine
+'''
+
+EXAMPLES = r'''
+- name: Get Podman system information
+  containers.podman.podman_system_info:
+
+- name: Get Podman system information into a variable
+  containers.podman.podman_system_info:
+  register: podman_info
+- name: Printing Podman System info
+  debug:
+    msg: "{{ podman_info['podman_system_info'] }}
```

Comment:
no ending double quote
```suggestion
    msg: "{{ podman_info['podman_system_info'] }}"
```